src/main.cpp

```diff
@@ -410,6 +474,24 @@ bool formValidator(iotwebconf::WebRequestWrapper* webRequestWrapper) {
     }
   }
 #if defined(EBUS_INTERNAL)
+  if (webRequestWrapper->arg(sntpServerParam.getId()).length() >
+      STRING_LEN - 1) {
+    String tmp = "max. ";
+    tmp += String(STRING_LEN);
```

Comment:
Maximum length of DNS name is 255 characters.
Please define macro DNS_LEN and use where applicable.